robottelo/test.py

```diff
@@ -238,11 +238,21 @@ def take_screenshot(self, testmethodname):
         testmethodname = "_".join(testmethodname)
         # Creates dir_structure depending upon the testmethodname.
         directory = testmethodname
-        if not os.path.exists("/tmp/screenshots/" + directory):
-            os.makedirs("/tmp/screenshots/" + directory)
-        self.browser.save_screenshot('/tmp/screenshots/{dir}/'
-                                     'screenshot-{date}.png'
-                                     .format(dir=directory, date=filename))
+        dir_structure = '{screenshots_dir}/{date}/{dir}'.format(
+            screenshots_dir=self.screenshots_dir,
+            date=datetime.now().strftime('%Y-%m-%d'),
+            dir=directory
+        )
+        if not os.path.exists(dir_structure):
+            os.makedirs(dir_structure)
+        self.browser.save_screenshot('{dir_structure}/screenshot-{dnt}.png'
+                                     .format(
+                                         dir_structure=dir_structure,
+                                         dnt=datetime.now().strftime(
+                                            '%Y-%m-%d_%H_%M_%S'
+                                         ),
+                                     )
+        )
```

Comment:
You can probably elimintate the Travis warnings with something like this:

    self.browser.save_screenshot(
        '{dir_structure}/screenshot-{dnt}.png'.format(
            dir_structure=dir_structure,
            dnt=datetime.now().strftime('%Y-%m-%d_%H_%M_%S'),
        )
    )